test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/BlockingAsyncRequestResponseBodyTest.java

```diff
@@ -86,4 +98,107 @@ public void blockingResponseTransformer_closeDoesNotThrow() throws IOException {
         ResponseInputStream<StreamingOutputOperationResponse> responseStream = responseFuture.join();
         responseStream.close();
     }
+
+    public void blockingOutputStreamWithoutExecutor_sendsRightValues() throws IOException {
```

Comment:
Missing `@Test`?

Reply:
Whoops, good catch. Lost it with all the rebasing involved splitting this into 3 commits.